### DISCUSSION TITLE
Rlink 2 screen rotation

### DISCUSSION THREAD

**Discussion by Serg797979** [cedricp/ddt4all#914] [Q&A]
Hello. Radio LANR14 megane 4. After unlocking and configuring, I can’t turn the display image to landscape mode. How can I do it. And maybe the firmware version affects this? Firmware 2.2.17

**Comment by AndyIreland** [comment]
could it be that this setting has to be changed here?:
![image](https://github.com/cedricp/ddt4all/assets/49172217/b7ff9256-431e-4c1e-afbe-d964857559a7)

**Comment by Serg797979** [comment]
I tried. Does not help. After configuration it was set to landscape

**Comment by matzel82** [comment]
Original Automatic configuration with clip

**Comment by AndyIreland** [comment]
> I tried. Does not help. After configuration it was set to landscape

did you press hard reset after you made the changes or tried locking car up for 20 minutes to go to sleep and then come back to car to see if landscape happened after reboot of the rlink?

I am not that into rlink2  and only ever done  work on the  rlink1 and never had an rlink2 , but with rlink1 to accept the changes with DDT4ALL I have to press 'hard reset' to reboot the rlink1 or press the home button 5 times in sucession or lock up car for 20mins and then go back later and the rlink1 reboots as I get into car and new changes are initiated 
Good luck

**Comment by Serg797979** [comment]
> Оригинальная автоматическая конфигурация с зажимом
The radio is not from this car. And my clip doesn't find the configuration file for it.

**Comment by Serg797979** [comment]
> > Я пытался. Не помогает. После настройки был установлен альбомный режим.
> 
> Вы нажимали полный сброс после внесения изменений или пытались запереть машину на 20 минут, чтобы заснуть, а затем вернулись в машину, чтобы посмотреть, произошел ли ландшафт после перезагрузки rlink?

I haven't tried it. I just pressed reset. Can hard reset help?

**Comment by AndyIreland** [comment]
> > > Я пытался. Не помогает. После настройки был установлен альбомный режим.
> > 
> > 
> > Вы нажимали полный сброс после внесения изменений или пытались запереть машину на 20 минут, чтобы заснуть, а затем вернулись в машину, чтобы посмотреть, произошел ли ландшафт после перезагрузки rlink?
> 
> I haven't tried it. I just pressed reset. Can hard reset help?

it does on my rlink1 - it reboots the rlink. **DO NOT whatever you do press 'SOFT RESET' !!**

**Comment by Serg797979** [comment]
> на моем rlink1 это происходит - он перезагружает rlink. **НЕ НАЖМАЙТЕ «SOFT RESET» !!**

Thank you. I will try.

**Comment by shrlnm** [comment]
@Serg797979 
https://www.drive2.ru/b/643019689764615344/

**Comment by Serg797979** [comment]
> @Serg797979 https://www.drive2.ru/b/643019689764615344/

Я этим способом и конфигурировал

**Comment by Serg797979** [comment]
Проблема решилась с помощью прошивки с версии 2.2 до версии 3.3
